lockbox-ios/Common/Resources/Constants.swift

```diff
@@ -52,7 +52,7 @@ struct Constant {
         static let unnamedEntry = NSLocalizedString("unnamed_entry", value: "unnamed entry", comment: "Placeholder text for when there is no entry name")
         static let username = NSLocalizedString("username", value: "Username", comment: "Section title text for username on the item detail screen")
         static let usernamePlaceholder = NSLocalizedString("username_placeholder", value: "(no username)", comment: "Placeholder text when there is no username")
-        static let unlockPlaceholder = NSLocalizedString("unlock_placeholder", value: "Unlock Firefox Lockbox", comment: "Placeholder text when the user's email is unavailable while unlocking Lockbox")
+        static let unlockPlaceholder = NSLocalizedString("unlock_placeholder", value: "Firefox Lockbox will use Touch ID to sign you in to the app.", comment: "Placeholder text when the user's email is unavailable while unlocking Lockbox, matches NSFaceIDUsageDescription")
```

Comment:
Ah yes, good catch thanks. 

Or on just a non-enrolled Touch ID device:


<img width="252" alt="screen shot 2018-05-14 at 12 34 03 pm" src="https://user-images.githubusercontent.com/49511/40016309-2d3e814a-5773-11e8-808f-7743da289023.png">

@changecourse What if this fallback just said "This will unlock the app." which should make sense in these contexts? (versus what it currently says which is "Unlock Firefox Lockbox")

Reply:
yeah, i think that makes sense for now.